q2_qemistree/tests/test_process_fingerprint.py

```diff
@@ -29,22 +30,30 @@ def setUp(self):
         self.goodcsi = qiime2.Artifact.load(os.path.join(THIS_DIR,
                                                          'data/csiFolder.qza'))
         properties_path = os.path.join(data, 'molecular_properties.csv')
-        self.properties = pd.read_table(properties_path, dtype=str)
+        self.properties = pd.read_csv(properties_path, dtype=str, sep='\t')
         self.properties.set_index('absoluteIndex', inplace=True)
 
     def test_fingerprintOut(self):
         msg = "Fingerprint file is empty!"
         with self.assertRaisesRegex(ValueError, msg):
             collate_fingerprint(self.emptycsi)
 
-    def test_featureMatch(self):
+    def test_featureMatch1(self):
         goodcsi = self.goodcsi.view(CSIDirFmt)
         tablefp = collate_fingerprint(goodcsi)
         features = load_table(self.featureTable)
         allfeatrs = set(features.ids(axis='observation'))
         fpfeatrs = set(tablefp.index)
         self.assertEqual(fpfeatrs <= allfeatrs, True)
 
+    def test_featureMatch2(self):
+        goodcsi = self.goodcsi.view(CSIDirFmt)
+        tablefp = collate_fingerprint(goodcsi)
+        smiles = get_feature_smiles(goodcsi, tablefp)
+        fpfeatrs = set(tablefp.index)
```

Comment:
I think these three lines could be replaced by using assert_frame_equals or assert_series_equals from Pandas' testing utilities.